client/components/Views/Report/Table.tsx

```diff
@@ -47,7 +49,15 @@ export default function ReportTable(props: ReportTableProps) {
                       fail: props.data[rowNumber].errors.has(innerIndex + 1),
                     })}
                   >
-                    <div className="cell-content">{value}</div>
+                    {value.length > 50 ? (
+                      <LightTooltip title={value}>
+                        <div className="cell-content" style={{ lineClamp: 1 }}>
+                          {truncate(value, { length: 50 })}
```

Comment:
it looks great @roll but I think it should be when it's less than 50. This is how i see tables, there is the ellipsis after around 17 characters. I would expect the tootltip to be visible for anything that is shorteded by the ellipsis. maybe is there a way of detecting how much of the text is visible?  what do you think @romicolman ? 
![image](https://github.com/user-attachments/assets/5e78d03c-487c-45e6-9eaf-78181539d9b1)

Reply:
Yea, we can reduce the visible size

Reply:
@roll maybe this can be addressed in a future issue. The problem I reported is different from what it is required on this ticket as Romina mentioned.